segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeepertier2/BookkeeperSegmentHandle.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.bookkeepertier2;
+
+import com.google.common.base.Preconditions;
+import io.pravega.segmentstore.storage.SegmentHandle;
+
+public class BookkeeperSegmentHandle implements SegmentHandle {
```

Comment:
package-private and not public. We don't want other classes to create random handles.

Reply:
Fixed.